src/components/map/Mapbox.vue

```diff
@@ -64,6 +111,14 @@ export default {
     navigateToEvent(event) {
       const { _id: id } = event;
       this.$router.push({ path: `/eventpage/${id}` });
+    },
+    updateQuery(payload) {
+      // console.log(payload);
```

Comment:
Try to **remove** console logs I think, makes everything cleaner

Reply:
Yep I'm gonna clean the code